### PR TITLE
Add support to use python build_ext -i command on windows

### PR DESCRIPTION
**Pull request by BrentHoltsclaw**
(assuming VS and WDK are installed)

### CODE REVIEW

File: chipsec_tools/compression/build.cmd [hunk 1]
```diff
@@ -16,27 +16,30 @@
 @REM #Contact information:
 @REM #chipsec@intel.com
 
+goto :main
+
+:set_buildvars
+@echo Building compression
+@echo off
+@REM # Need to set EDK_TOOLS_PATH and BASE_TOOLS_PATH to current directory and HOST_ARCH
```
Comment (NicholasArmour) [inline chipsec_tools/compression/build.cmd]: Does HOST_ARCH still need to be set?

File: setup.py [hunk 2]
```diff
@@ -139,6 +139,28 @@ def _build_darwin_compression(self):
         for exe in exes:
             self.copy_file(os.path.join(build_exe,"bin",exe),dst)
 
+    def _build_win_driver(self):
+        log.info("building the windows driver")
+        build_driver = os.path.join("drivers", "win7")
+        cur_dir = os.getcwd()
+        os.chdir(build_driver)
+        print(os.getcwd())
```
Comment (NicholasArmour) [inline setup.py]: debug print? do we need this?

File: chipsec_tools/compression/build.cmd (hunk 1 above)
Comment (BrentHoltsclaw) [reply chipsec_tools/compression/build.cmd]: should be 32 bit to compile, will remove the comment

File: setup.py (hunk 2 above)
Comment (BrentHoltsclaw) [reply setup.py]: good catch removing

**Pull request merged**